src/adapt/vgen.js

```diff
@@ -1205,6 +1210,59 @@ adapt.vgen.ViewFactory.prototype.preprocessElementStyle = function(computedStyle
 
 /**
  * @private
+ * @param {Element} element
+ * @param {adapt.cssstyler.AbstractStyler} styler
+ */
+adapt.vgen.ViewFactory.prototype.findAndProcessRepeatingElements = function(element, styler) {
+    for (var child = element.firstChild; child; child = child.nextSibling) {
+        if (child.nodeType !== 1) continue;
+        var computedStyle = {};
+        var elementStyle = styler.getStyle(/** @type {Element}*/ (child), false);
+        this.computeStyle(this.nodeContext.vertical, elementStyle, computedStyle);
+        var processRepeateOnBreak = this.processRepeateOnBreak(computedStyle);
+        if (!processRepeateOnBreak) continue;
+
+        if (this.nodeContext.formattingContext instanceof vivliostyle.repetitiveelements.RepetitiveElementsOwnerFormattingContext
+            && !this.nodeContext.formattingContext.isInherited(this.nodeContext)) {
+            return;
+        }
+
+        var parent = this.nodeContext.parent;
+        var parentFormattingContext = parent && parent.formattingContext;
+        this.nodeContext.formattingContext =
+            new vivliostyle.repetitiveelements.RepetitiveElementsOwnerFormattingContext(
+                parentFormattingContext, /** @type {!Element}*/ (this.nodeContext.sourceNode));
+        this.nodeContext.formattingContext.initializeRepetitiveElements(this.nodeContext.vertical);
+        return;
+    }
+};
+
+/**
+ * @private
+ * @param {!Object.<string,adapt.css.Val>} computedStyle
+ */
+adapt.vgen.ViewFactory.prototype.processRepeateOnBreak = function(computedStyle) {
```

Comment:
Fix it in [this commit](https://github.com/vivliostyle/vivliostyle.js/pull/323/commits/9229639dd75eedc9529afa5708b180a40d54f052).